api/app/endpoints/wms.py

```diff
@@ -65,14 +83,15 @@ def parse_size(params) -> Size:
             "correctly from the list of parameters ",
         )
     if (height * width) > current_app.config["WMS"]["MAX_SIZE"]:
-        abort(400, "Total size is bigger than the maximaml allowed size")
+        abort(400, "Total size is bigger than the maximal allowed size")
     return Size(width=width, height=height)
 
 
 Position = namedtuple("Position", ("x", "y"))
 
 
 def parse_position(params) -> Size:
```

Comment:
The type of the variable returned is not matching the type the function expects??